thoth/messaging/message_factory.py

```diff
@@ -19,9 +19,28 @@
 """This is Thoth Messaging module for message_factory."""
 
 import faust
+
 from typing import Tuple
+from typing import Dict  # noqa
```

Comment:
why the no quality assurance(noqa) comment?

Reply:
because it is imported but unused. It may be used later on during the construction of the `MessageContents` class